### DISCUSSION TITLE
105 empty prompts for 105 styles.

### DISCUSSION THREAD

**Discussion by ptits** [lllyasviel/Fooocus#249] [General]
I was curious what is behind the styles, so I rendered (with some auto-magic) 105 images with empty prompts to see what is "addition" of style. Seed = 0.
Then I fed html-log with images to chatGPT Code Interpreter and asked it to get style names and rename files to styles. Then I asked it to add frame around each image and text labels below and make collage of them. Here is final collage from Fooocus and chatGPT.
Funny to look at some pictures.
![vertical_collage1](https://github.com/lllyasviel/Fooocus/assets/13587736/e7543479-e109-40a8-b5f1-8c2569b11fd2)
![vertical_collage2](https://github.com/lllyasviel/Fooocus/assets/13587736/b8c4ff4b-0753-4f5e-86b4-2a9f908e96eb)

**Comment by barepixels** [comment]
What is this awesome "auto-magic"? :-)  Oh man I would love to run the same prompt through all styles or a custom list of styles

**Reply by ptits** [reply]
I made it brute-force simple))
Used Autohotkey script to push buttons like monkey and avoid python scripting

**Comment by ArnorWing** [comment]
Can i have your permission to use your image in a tutorial video for YT?

**Reply by ptits** [reply]
Sure.
Be nice if you point the source of images as https://t.me/cgevent/6355

**Reply by ArnorWing** [reply]
I was going to link to this thread, So edit your 1st post if you can with your TG link. That ok?
I will delete my post afterwards.

**Reply by ptits** [reply]
I prefer you can make link to telegram posts where I explained all in details.
https://t.me/cgevent/6362
https://t.me/cgevent/6361

**Comment by robinwang25** [comment]
Unbelievable, It's so unbelievable

**Comment by aimarxjg** [comment]
I prefer the girl list you made❤️
